jre_emul/Makefile

```diff
@@ -206,8 +206,8 @@ dist_includes: $(BUILD_DIR)/.jre_dist_includes
 $(BUILD_DIR)/.jre_dist_includes: $(PUBLIC_TRANSLATED_HEADERS:%=$(GEN_OBJC_DIR)/%) \
     $(PUBLIC_EMULATION_HEADERS:%=$(EMULATION_CLASS_DIR)/%) | $(DIST_INCLUDE_DIR)
 	@echo copying jre_emul header files
-	@(cd $(EMULATION_CLASS_DIR) && tar cf - \
-	    $(PUBLIC_EMULATION_HEADERS:$(EMULATION_CLASS_DIR)/%=%)) \
+	@cd $(EMULATION_CLASS_DIR) && tar cf - \
```

Comment:
Well, that would certainly make it easier to read the makefiles... but
that's in a lot of places :-/

On Wed, Feb 3, 2016 at 12:26 PM Keith Stanger <notifications@github.com>
wrote:

> In jre_emul/Makefile
> <https://github.com/google/j2objc/pull/700#discussion_r51764103>:
>
> > @@ -206,8 +206,8 @@ dist_includes: $(BUILD_DIR)/.jre_dist_includes
> >  $(BUILD_DIR)/.jre_dist_includes: $(PUBLIC_TRANSLATED_HEADERS:%=$(GEN_OBJC_DIR)/%) \
> >      $(PUBLIC_EMULATION_HEADERS:%=$(EMULATION_CLASS_DIR)/%) | $(DIST_INCLUDE_DIR)
> >  	@echo copying jre_emul header files
> > -	@(cd $(EMULATION_CLASS_DIR) && tar cf - \
> > -	    $(PUBLIC_EMULATION_HEADERS:$(EMULATION_CLASS_DIR)/%=%)) \
> > +	@cd $(EMULATION_CLASS_DIR) && tar cf - \
>
> It's possible that the subshell isn't necessary. According to this a
> subshell is implied by the pipe:
>
> http://stackoverflow.com/questions/5760640/left-side-of-pipe-is-the-subshell
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/j2objc/pull/700/files#r51764103>.
>